src/intents.js

```diff
@@ -32,8 +32,10 @@ function injectService (url, element, intent, data, onReadyCallback) {
   // if callback provided for when iframe is loaded
   if (typeof onReadyCallback === 'function') iframe.onload = onReadyCallback
   iframe.setAttribute('src', url)
+  title && iframe.setAttribute('title', title)
   iframe.classList.add(intentClass)
   element.appendChild(iframe)
+  element.getElementsByClassName('coz-intent')[0].focus()
```

Comment:
I think it's our iframe, so maybe we can just do:
```js
iframe.focus()
```
And avoid to query the DOM (Also, `cozy-intent` is already in a variable).

What do you think ?